### DISCUSSION TITLE
mac m1 启动PPOCRLable 失败

### DISCUSSION THREAD

**Discussion by sirwangshuai** [PaddlePaddle/PaddleOCR#12619] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：M1 mac 版本13.0.1
- 版本号/Version：Paddle：2.3.1  PaddleOCR：2.7  问题相关组件/Related components：PPOCRLable
- 运行指令/Command Code：PPOCRLabel --lang ch
- 完整报错/Complete Error Message：
- Traceback (most recent call last):
  File "/Users/wangshuai/mambaforge/envs/paddle/bin/PPOCRLabel", line 5, in <module>
    from PPOCRLabel.PPOCRLabel import main
  File "/Users/wangshuai/mambaforge/envs/paddle/lib/python3.9/site-packages/PPOCRLabel/PPOCRLabel.py", line 27, in <module>
    from PyQt5.QtCore import QSize, Qt, QPoint, QByteArray, QTimer, QFileInfo, QPointF, QProcess
ImportError: dlopen(/Users/wangshuai/mambaforge/envs/paddle/lib/python3.9/site-packages/PyQt5/QtCore.abi3.so, 0x0002): Symbol not found: __ZTVNSt3__13pmr25monotonic_buffer_resourceE
  Referenced from: <645DDC2C-F655-324A-BE87-40804F3AC471> /Users/wangshuai/mambaforge/envs/paddle/lib/python3.9/site-packages/PyQt5/Qt5/lib/QtCore.framework/Versions/5/QtCore
  Expected in:     <AE8EA4B2-B8FD-33E3-A0AF-0B7485AB97C0> /usr/lib/libc++.1.dylib

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by g39088902** [comment]
是否安装了PyQt5?

**Comment by sirwangshuai** [comment]
> 是否安装了PyQt5?

已安装
